miniwdl_aws/batch_job.py

```diff
@@ -179,10 +180,14 @@ def _run(self, logger, terminating, command):
         Run task
         """
         try:
+            if os.environ.get("MINIWDL__AWS__BATCH_CLIENT_RETRIES") is not None:
+                aws_batch_client_retries = json.loads(os.environ["MINIWDL__AWS__BATCH_CLIENT_RETRIES"])
+            else:
+                aws_batch_client_retries = self.cfg["batch_client"].get_dict("default_retries")
```

Comment:
The configuration loader (`self.cfg`) should honor the environment variable automatically, i.e. it shouldn't be necessary to explicitly check the environment variable above. [details](https://github.com/chanzuckerberg/miniwdl/blob/eedf2e04642f82b4337033dda21b1dce8eadcac0/WDL/runtime/config.py#L54-L72)